### PR TITLE
add contributing.md, based on version in xdem

### PR DESCRIPTION
**Pull request by atedstone**
Addresses #34

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Rights
+The license (see LICENSE) applies to all contributions.
+
+## Issue Conventions
+When submitting bugs, please filll out the bug report template in full.
+
+Please search existing issues, open and closed, before creating a new one.
+
+## Git conventions
+Work on features should be made on a fork of `geoutils` and submitted as a pull request (PR) to main or a relevant branch.
```
Comment (erikmannerfelt) [inline CONTRIBUTING.md:12]: Great! Maybe just change "main" to "master" here to be consistent with the primary branch name.

**Review by adehecq — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# How to contribute
+
+## Rights
+The license (see LICENSE) applies to all contributions.
+
+## Issue Conventions
+When submitting bugs, please filll out the bug report template in full.
```
Comment (jlandmann) [inline CONTRIBUTING.md:7]: Hi Andrew! Thanks for the great work and sorry for commenting out of nothing...i've just seen by coincidence that here is a little typo ;-)

File: CONTRIBUTING.md (hunk 1 above)
Comment (erikmannerfelt) [reply CONTRIBUTING.md]: Fixed this in 1e0c370

File: CONTRIBUTING.md (hunk 2 above)
Comment (erikmannerfelt) [reply CONTRIBUTING.md]: Fixed in 1e0c370. Thanks, @jlandmann!

**Pull request merged**